feat: 2674 discourse plugin

vince0656 opened the pull request:
<!-- Use this template by filling in information and copying and pasting relevant items out of the HTML comments. -->

# Relates to

https://github.com/elizaOS/eliza/issues/2674

# Risks

Low - it pulls post data from a known Discourse instance and uses axios.

# Background

## What does this PR do?

Adds a discourse plugin

## What kind of change is this?

Features (non-breaking change which adds functionality)


# Documentation changes needed?
My changes do not require a change to the project documentation.

<!-- Please show how you tested the PR. This will really help if the PR needs to be retested and probably help the PR get merged quicker. -->

# Testing

`pnpm run test`

guspan-tanadi committed:
chore(packages/clients): include section links

docs(packages/adapters): section links
chore(packages/plugins): implementation links Allora

AIFlowML committed:
Merge branch 'develop' into fix-plugin-dexscreener

AIFlowML committed:
Fixed all the issues but 1 any

AIFlowML committed:
Merge branch 'develop' into fix-plugin-binance

odilitime committed:
Merge pull request #3089 from AIFlowML/fix-plugin-binance

fix: plugin-binance lint

AIFlowML committed:
fix: plugin-birdeye (#3088)

* Fixed almost all issues bt could not taouch the types.ts

* Fixed all the issues

AIFlowML committed:
Fixed almost all the issues (#3087)

Co-authored-by: Odilitime <janesmith@airmail.cc>

AIFlowML committed:
fix: plugin-Chainbase lint (#3085)

* Fixed almost all issues bt could not taouch the types.ts

* Fixed all the issue but the template file.

AIFlowML committed:
fix: plugin-coingecko lint (#3084)

* Fixed almost all issues bt could not taouch the types.ts

* Fixed all the issues

AIFlowML committed:
fix: plugin-coinmarketcap lint (#3082)

* Fixed almost all issues bt could not taouch the types.ts

* Fixed few minor issues

AIFlowML committed:
Fixed almost all issues bt could not taouch the types.ts (#3080)

AIFlowML committed:
Fixed all issues but a file (#3077)

AIFlowML committed:
Fixed all the issues (#3076)

AIFlowML committed:
Fixed almost all the issues (#3086)

Co-authored-by: Odilitime <janesmith@airmail.cc>

odilitime committed:
Merge pull request #3026 from PhyByte/develop

fix: Rename file with typo in MVX plugin

odilitime committed:
Merge branch 'develop' into develop

odilitime committed:
Merge pull request #3081 from EmanHerawy/develop

docs: Add Arabic translation for README

odilitime committed:
Merge pull request #3075 from AIFlowML/fix-plugin-cronoszkevm

fix: plugin-cronoszkevm lint

odilitime committed:
Merge pull request #3074 from AIFlowML/fix-plugin-dcap

fix: plugin-dcap lint

AIFlowML committed:
Fixed all issues, i have an shadow error on packages/plugin-b2/src/providers/index.ts line 103 it appear and disapper (#3091)

AIFlowML committed:
Fixed all the issues (#3073)

odilitime committed:
Merge branch 'develop' into fix-plugin-dexscreener

odilitime committed:
Merge pull request #3069 from AIFlowML/fix-plugin-dexscreener

fix: plugin-dexscreener lint

btspoony committed:
fix: the erroneous corrections caused by biome. (#3052)

- fix the sample serviceType is undefined
- fix erroneous corrections in di plugin caused by biome.
- add Parameter Decorators support in biome

odilitime committed:
Merge branch 'develop' into fix-plugin-di

odilitime committed:
Merge pull request #3066 from AIFlowML/fix-plugin-di

fix: fix-plugin-di

AIFlowML committed:
Fixed all the issues (#3092)

Co-authored-by: Odilitime <janesmith@airmail.cc>

AIFlowML committed:
Fixed all the errors (#3063)

AIFlowML committed:
Fixed all the issues (#3065)

Co-authored-by: Odilitime <janesmith@airmail.cc>

AIFlowML committed:
Fixed all typings and wrong variables (#3061)

MichaelDeng03 committed:
feat: 3049 add demo api access coingecko (#3050)

* Update getPrice.ts to use headerKey in coin price request.

* Update getMarkets.ts to check api config for api headerkey

AIFlowML committed:
Fixed all the issues and added biome (#3044)

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

AIFlowML committed:
fix: plugin-avail  (#3094)

* Fixed all the issues

* fix: add biome configuration and fix linting issues in plugin-avail

odilitime committed:
Merge pull request #3005 from guspan-tanadi/adapterslinks

docs(packages/adapters): navigate section links

john-xina-p88 committed:
chore: readme

john-xina-p88 committed:
feat: sequentially cancel orders

john-xina-p88 committed:
Merge branch 'develop' into feature/plugin-desk-exchange

john-xina-p88 committed:
Merge pull request #1 from HMXOrg/feature/plugin-desk-exchange

feat: plugin-desk-exchange

samarth30 committed:
fix: fixed a type issue due to which build was failing

samarth30 committed:
fix: fixed a type issue due to which build was failing

AIFlowML committed:
fix: add biome configuration and fix linting issues in plugin-autonome

tcm390 committed:
Merge pull request #3097 from elizaOS/fix/plugin-email-automation

fix: fixed build error in plugin-email-automation

AIFlowML committed:
Great code i just added the Biome

YoungPhlo committed:
Merge branch 'develop' into docs/community-streams

john-xina-p88 committed:
feat: use bigint to prevent overflow

john-xina-p88 committed:
Merge branch 'develop' into develop

YoungPhlo committed:
docs: Add "What Did You Get Done This Week? 11" notes

AIFlowML committed:
Fixed all the issues this is the commit to keep for arthers

AIFlowML committed:
Fixed all the issues in Aptos and aded Biome

AIFlowML committed:
Merge branch 'develop' into fix-plugin-asterai-clean

AIFlowML committed:
Fixed the initial error in wallet and multiple other typings. I left 2 because not possibler to restore them fast. (#3010)

Co-authored-by: tcm390 <60634884+tcm390@users.noreply.github.com>

tcm390 committed:
Merge branch 'develop' into fix-plugin-asterai-clean

tcm390 committed:
Merge pull request #3101 from AIFlowML/fix-plugin-asterai-clean

fix: plugin-asterai

tcm390 committed:
Merge branch 'develop' into fix-plugin-autonome-clean

tcm390 committed:
Merge pull request #3098 from AIFlowML/fix-plugin-autonome-clean

fix: plugin-autonome-v1

tcm390 committed:
Merge branch 'develop' into fix-plugin-arthera-biome-clean

tcm390 committed:
Merge pull request #3102 from AIFlowML/fix-plugin-arthera-biome-clean

fix: plugin-arthera-biome

tcm390 committed:
Merge branch 'develop' into fix-plugin-aptos

tcm390 committed:
Merge pull request #3104 from AIFlowML/fix-plugin-aptos

fix: plugin-aptos

tcm390 committed:
Merge branch 'develop' into docs/community-streams

tcm390 committed:
Merge pull request #3103 from YoungPhlo/docs/community-streams

docs: Add "What Did You Get Done This Week? 11" notes

AIFlowML committed:
Investigated and added the peoper ether to the configuration it was missing and fixed all the code

AIFlowML committed:
Merge branch 'develop' into fix-plugin-apro

AIFlowML committed:
Fixed all the issues and retouch a bit te code in in Anyone Plugin

AIFlowML committed:
Merge branch 'develop' into fix-plugin-anyone

jonathangus committed:
Update Dockerfile

vipocenka committed:
typo index.ts

vipocenka committed:
fix typo api.ts

vipocenka committed:
fix typo GoplusSecurityService.ts

vipocenka committed:
fix typo README.md

maxime committed:
fix: fixing the error parsing json when an array is a value in a JSON object

maxime committed:
Merge branch 'develop' into bug/json_parsing_of_arrays_as_value

john-xina-p88 committed:
Merge branch 'develop' into develop

tcm390 committed:
fix scroll to bottom

ai16z-demirix committed:
Merge branch 'develop' into tests/plugin-chainbase

ai16z-demirix committed:
Update package.json

tcm390 committed:
Merge pull request #3113 from maxime/bug/json_parsing_of_arrays_as_value

fix: fixing the error parsing json when an array is a value in a JSON…

tcm390 committed:
Merge branch 'develop' into fix/typos

tcm390 committed:
Merge branch 'develop' into tcm-fix-client-scroll

wtfsayo committed:
updated gitignore + removed build files (#3117)

ai16z-demirix committed:
Merge branch 'develop' into tests/plugin-chainbase

shakkernerd committed:
Merge branch 'develop' into tcm-fix-client-scroll

tcm390 committed:
Merge pull request #3115 from elizaOS/tcm-fix-client-scroll

fix: client scroll to bottom

shakkernerd committed:
Merge branch 'develop' into fix-plugin-apro

shakkernerd committed:
Merge pull request #3106 from AIFlowML/fix-plugin-apro

fix: plugin-apro

ai16z-demirix committed:
Merge branch 'develop' into tests/plugin-chainbase

shakkernerd committed:
Merge pull request #3072 from ai16z-demirix/tests/plugin-chainbase

feat (chore): plugin-chainbase test config and coverage

ai16z-demirix committed:
feat (chore): plugin bittensor test config and coverage (#3064)

* plugin-bittensor: package json

* plugin-bittensor: vitest config

* plugin-bittensor: actions, sn34 test

* plugin-bittensor: fact tests

* Update fact.test.ts

* Update package.json

fixing conflict

* Update package.json

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

ai16z-demirix committed:
feat (chore): plugin-bootstrap: test config and test coverage (#3068)

* Update fact.test.ts

* Update goal.test.ts

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

shakkernerd committed:
Merge branch 'develop' into fix/typos

shakkernerd committed:
Merge branch 'develop' into docker-fix

shakkernerd committed:
Merge pull request #3110 from jonathangus/docker-fix

fix: docker build command by skipping eslint files

Commit:
chore(plugin-chainbase): add @biomejs/biome dev dependency

shakkernerd committed:
fix: export dexScreenerPlugin

shakkernerd committed:
Merge pull request #3120 from elizaOS/fix/plugin-dexscreener

fix: export dexScreenerPlugin

shakkernerd committed:
Merge branch 'develop' into fix/plugin-chainbase

shakkernerd committed:
Merge pull request #3118 from akugone/fix/plugin-chainbase

fix:chore(plugin-chainbase): add missing comma in package.json

alexpaden committed:
save imageUrls for outbound tweets/messages

interactions.ts handles inbound
utils.ts handles outbound

shakkernerd committed:
fix: remove duplicate litPlugin import

shakkernerd committed:
Merge pull request #3121 from elizaOS/fix/agent_duplicates

fix: remove duplicate litPlugin import

0xCardiE committed:
add provider, action and template for pools per network

0xCardiE committed:
change names

odilitime committed:
Merge pull request #2361 from elizaOS/develop

chore: dev => main 0.1.9

tcm390 committed:
remove duplicated plugins

john-xina-p88 committed:
Merge branch 'develop' into develop

vipocenka committed:
Update api.ts

tcm390 committed:
Merge branch 'develop' into fix/typos

tcm390 committed:
Merge pull request #3111 from vipocenka/fix/typos

fix: typos in multiple files

v1xingyue committed:
some fix

v1xingyue committed:
Merge branch 'main' into some-fix

tcm390 committed:
Merge branch 'develop' into feature/twitter-messages-image-outbound

tcm390 committed:
Merge pull request #3122 from alexpaden/feature/twitter-messages-image-outbound

feat: save imageUrls for outbound tweets/messages

HashWarlock committed:
fix: upgrade openai and vercel ai packages to fix o1 errors

lalalune committed:
Merge pull request #3146 from elizaOS/fix/openai-o1-fix

fix: upgrade openai and vercel ai packages to fix o1 errors

You-saku committed:
add unwritten text and fix typo in README_JA.md

v1xingyue committed:
update pnpm version in dockerfile

Marcofann committed:
fix ADRESS_SECURITY_CHECK in GoplusSecurityService.ts

Marcofann committed:
fix ADRESS_SECURITY_CHECK in index.ts

Marcofann committed:
fix ADRESS_SECURITY_CHECK in GoPlusManage.ts

Marcofann committed:
fix ADRESS_SECURITY_CHECK in README.md

strmfos committed:
Update chat_2024-12-07.md

strmfos committed:
Update chat_2024-10-27.md

PixelPil0t1 committed:
Update image.yaml

PixelPil0t1 committed:
Update pr.yaml

AIFlowML committed:
Fixed all the issues and added biome

AIFlowML committed:
Merge branch 'develop' into fix-plugin-allors

0xCardiE committed:
Merge branch 'develop' into feat/coingecko_advanced_more_endpoints

AIFlowML committed:
Fixed multiple issues and aded biome.

AIFlowML committed:
Merge branch 'main' into fix-plugin-akash-v1

AIFlowML committed:
Fixed the typing and added biome

AIFlowML committed:
Merge branch 'main' into fix-plugin-agentkit

AIFlowML committed:
fixed all the issues but deployTokenAction.ts for the ANY and the result need probaly a refactoring.

AIFlowML committed:
Merge branch 'develop' into fix-plugin-abstract

AIFlowML committed:
Merge branch 'develop' into fix-plugin-agentkit

0xCardiE committed:
add new pools by network

AIFlowML committed:
Fixed all the issues and added biome

AIFlowML committed:
Merge branch 'develop' into fix-plugin-3g-gen

AIFlowML committed:
Fixed all the issues and added Biome

AIFlowML committed:
Merge branch 'develop' into fix-plugin-0x

0xCardiE committed:
add mappings from provider

AIFlowML committed:
Fixed all the issues and added biome. Only issues still in place in upload.ts i had to add an any to overcome the issue on the line 370 const flowContract = getFlowContract(runtime.getSetting(ZEROG_FLOW_ADDRESS), signer as any);

AIFlowML committed:
Merge branch 'develop' into fix-plugin-0g

AIFlowML committed:
Added biome to - plugin-zksync-era - plugin-zilliqa - plugin-zerion - plugin-whatsapp

AIFlowML committed:
Merge branch 'develop' into fix-multi-biome-01

AIFlowML committed:
Add Biome linting and formatting configuration to the following plugins:
- plugin-web-search
- plugin-video-generation
- plugin-udio
- plugin-twitter

shakkernerd committed:
Merge pull request #3180 from AIFlowML/fix-multi-biome-01

fix: multi-biome-01

shakkernerd committed:
Merge branch 'develop' into fix-plugin-0g

AIFlowML committed:
Merge branch 'develop' into fix-multi-biome-02

shakkernerd committed:
Merge pull request #3179 from AIFlowML/fix-plugin-0g

fix: plugin-0g

shakkernerd committed:
Merge branch 'develop' into fix-multi-biome-02

shakkernerd committed:
Merge pull request #3181 from AIFlowML/fix-multi-biome-02

fix: multi-biome-02

shakkernerd committed:
chore: remove multiple tsup package

shakkernerd committed:
Merge branch 'develop' into fix-plugin-0x

shakkernerd committed:
Merge pull request #3178 from AIFlowML/fix-plugin-0x

fix: plugin-0x

shakkernerd committed:
Merge branch 'develop' into fix-plugin-3g-gen

shakkernerd committed:
Merge pull request #3175 from AIFlowML/fix-plugin-3g-gen

fix: plugin-3g-generation

shakkernerd committed:
Merge branch 'develop' into fix-plugin-abstract

shakkernerd committed:
Merge pull request #3174 from AIFlowML/fix-plugin-abstract

fix: plugin-abstract

shakkernerd committed:
Merge branch 'develop' into fix-plugin-agentkit

shakkernerd committed:
Merge pull request #3172 from AIFlowML/fix-plugin-agentkit

fix: plugin-agentkit

shakkernerd committed:
Merge branch 'develop' into fix-plugin-akash-v1

shakkernerd committed:
Merge pull request #3171 from AIFlowML/fix-plugin-akash-v1

fix: plugin-akash

shakkernerd committed:
Merge branch 'develop' into fix-plugin-allors

shakkernerd committed:
Merge pull request #3169 from AIFlowML/fix-plugin-allors

fix: plugin-allora

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #3165 from comfsrt/main

docs: fix typos in .md

AIFlowML committed:
Add Biome linting and formatting configuration to the following plugins:
- plugin-ton
- plugin-trikon
- plugin-trustdb
- plugin-tts

Changes include:
- Add @biomejs/biome as devDependency
- Add lint and format scripts
- Replace existing ESLint configurations with Biome
- Maintain existing build and test scripts

ai16z-demirix committed:
feat (chore): plugin-coinmarketcap (#3134)

* plugin-coinmarketcap: vitest config

* plugin-coinmarketcap: package json

* plugin-coinmarketcap: test setup

* plugin-coinmarketcap: get price tests

* plugin-coinmarketcap: get price service tests

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

ai16z-demirix committed:
feat (chore): plugin-coingecko test config and coverage (#3124)

* plugin-coingecko: package json

* plugin-coingecko: vitest config

* plugin-coingecko: get price tests

* plugin-coingecko: test setup

* plugin-coingecko: get trending tests

* plugin-coingecko: top gainers losers tests

* plugin-coingecko: get market tests

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

AIFlowML committed:
Add Biome linting and formatting configuration to the following plugins:
- plugin-thirdweb
- plugin-tee-verifiable-log
- plugin-tee-marlin
- plugin-tee-log

AIFlowML committed:
Add Biome linting and formatting configuration to the following plugins:
- plugin-tee
- plugin-suno
- plugin-sui
- plugin-story

AIFlowML committed:
Add Biome linting and formatting configuration to the following plugins:
- plugin-starknet
- plugin-stargaze
- plugin-squid-router
- plugin-spheron

AIFlowML committed:
Add Biome linting and formatting configuration to the following plugins:
- plugin-solana-v2
- plugin-solana-agent-kit
- plugin-solana
- plugin-sgx

AIFlowML committed:
Merge branch 'develop' into fix-multi-biome-07

tcm390 committed:
Merge branch 'main' into tcm/remove-duplicated-plugins

tcm390 committed:
Merge branch 'develop' into tcm/remove-duplicated-plugins

tcm390 committed:
Merge pull request #3126 from elizaOS/tcm/remove-duplicated-plugins

fix: remove duplicated plugins

tcm390 committed:
update provider-utils

tcm390 committed:
Merge branch 'develop' into tcm-update-provider-utils

tcm390 committed:
Merge pull request #3189 from elizaOS/tcm-update-provider-utils

fix: update provider-utils

tcm390 committed:
extract attribute from raw text instead of normalized json

tcm390 committed:
Merge pull request #3190 from elizaOS/tcm-correct-extract

fix: extract attribute from raw text instead of normalized json

samarth30 committed:
Merge branch 'develop' into fix-multi-biome-07

samarth30 committed:
Merge branch 'develop' into feat/coingecko_advanced_more_endpoints

john-xina-p88 committed:
Merge branch 'develop' into develop

samarth30 committed:
Merge pull request #3170 from 0xCardinalError/feat/coingecko_advanced_more_endpoints

feat: coingecko advanced -  various pools by network

samarth30 reviewed (changes requested):
can you add required variables in env.example.

samarth30 committed:
Merge pull request #3045 from edwin-finance/plugin-edwin

feat: Add edwin plugin to eliza

samarth30 committed:
Merge pull request #3096 from HMXOrg/develop

feat: plugin desk exchange

maxime committed:
fix: make slack client able to download and upload attachments

nilaysarma committed:
Update Twitter to X (Twitter)

Since Twitter is now rebranded to X, so I have changes made changes in the README file. It is now "X (Twitter)" and I kept the Twitter keyword since people still use the word Twitter.

tcm390 committed:
Merge branch 'develop' into 3196--Update-Twitter-to-X

tcm390 committed:
Merge pull request #3198 from nilaysarma/3196--Update-Twitter-to-X

docs: Update Twitter to X (Twitter)

tcm390 committed:
Merge branch 'develop' into fix/slack_download_upload_attachments

Marcofann committed:
Update index.ts

shakkernerd committed:
Merge branch 'develop' into fix-multi-biome-07

shakkernerd committed:
Merge pull request #3186 from AIFlowML/fix-multi-biome-07

chore: add Biome configuration to Solana ecosystem plugins 07

gap-editor committed:
typo chat_2024-12-01.md

solono - solana

gap-editor committed:
typo chat_2024-12-03.md

vides - videos

gap-editor committed:
Update chat_2024-10-30.md

Docker composes - Docker Compose

shakkernerd committed:
Merge branch 'develop' into fix-plugin-anyone

shakkernerd committed:
Merge pull request #3107 from AIFlowML/fix-plugin-anyone

fix: plugin-anyone

YoungPhlo committed:
Merge branch 'develop' into docs/contributor-meetings

rferrari committed:
fix: think tag from venice

YoungPhlo committed:
docs: Add weekly contributor meeting notes (01-21 and 01-28)

gap-editor committed:
Update chat_2024-11-23.md

gap-editor committed:
typo chat_2024-12-02.md

YoungPhlo committed:
capitalized ElizaOS (for graphite)

Danyylka committed:
Update faq.md

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #3207 from Danyylka/main

docs: Update faq.md

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #3206 from gap-editor/main

chore: Fix Typos and Improve Consistency in Community Chat Logs

shakkernerd committed:
Merge branch 'develop' into docs/contributor-meetings

shakkernerd committed:
Merge pull request #3204 from YoungPhlo/docs/contributor-meetings

docs: Add weekly contributor meeting notes

shakkernerd committed:
Merge branch 'develop' into develop

shakkernerd committed:
Merge pull request #3203 from rferrari/develop

fix: think tag from venice

shakkernerd committed:
Merge branch 'develop' into fix/slack_download_upload_attachments

shakkernerd committed:
Merge pull request #3194 from maxime/fix/slack_download_upload_attachments

fix: Slack download upload attachments

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #3166 from PixelPil0t1/main

chore: Update GitHub Actions workflows and documentation

shakkernerd committed:
Merge branch 'develop' into fix/fix

shakkernerd committed:
Merge pull request #3164 from Marcofann/fix/fix

chore: Standardization of Security Check Identifiers across GoPlus plugin

shakkernerd committed:
Merge branch 'develop' into docker-pnpm-version

shakkernerd committed:
Merge pull request #3158 from v1xingyue/docker-pnpm-version

fix: Update pnpm version during Docker build

shakkernerd committed:
Merge branch 'develop' into fix-readme-ja

shakkernerd committed:
Merge pull request #3153 from You-saku/fix-readme-ja

Fix README_JA.md (add unwritten text and fix typo)

shakkernerd committed:
Merge branch 'develop' into some-fix

shakkernerd committed:
chore: conditional logging for server starting on a different port

shakkernerd committed:
Merge pull request #3141 from v1xingyue/some-fix

chore: some fix after v0.1.9

silasneo committed:
feat: Trump character but tweets in Spanish by default (#3119)

* Update README.md

small note at the top of readme to highlight contribution via this fork

* custom character file

* Trump in Latin American Spanish

Modified Trump character file to have the agent speak in Latin Spanish by default.
The agent will respond in any language if specifically asked to respond in that language in the user input.
This file can serve as a template for creating agents who speak a specific language by default for a given language audience.

* Update README.md

* Delete characters/uncleugo4.character.json

* Adding spanish_trump.character.json

Many people are not native English speakers & a template was needed to make it easier for anyone to create an agent that speaks other languages by default. 
Modified Trump character file to have the agent speak in Latin Spanish by default. 
This file can serve as a template for creating agents who speak a specific language by default for a given language audience.
The agent will respond in any language if specifically asked to respond in that language in the user input.

* Update spanish_trump.character.json

minor correction for duplicated adjectives

* Update spanish_trump.character.json

* removed duplicate adjectives

* Update package.json

* Update spanish_trump.character.json

Removed some English in bio for consistency

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

azep-ninja committed:
optimizations & fixes.

elpulpo0 committed:
feat: Add CREATE_POOL action in MultiversX plugin

Fallengirl committed:
Update chat_2024-11-17.md

elpulpo0 committed:
feat: update constants in MultiversX plugin

mikirov committed:
feat: nft actions

odilitime committed:
Merge pull request #3211 from mikirov/feat/ton-plugin-nft-management

feat: TON Plugin: NFT collection, item creation, metadata change and transfer actions

odilitime committed:
Merge pull request #3210 from Fallengirl/patch-1

docs: rename chat_2024-11-17.md

odilitime committed:
Merge pull request #3209 from elpulpo0/develop

feat: Add CREATE_POOL action in MultiversX plugin

odilitime committed:
Merge branch 'develop' into fix-quickintel-plugin

tcm390 committed:
remove duplicated dependencies

tcm390 committed:
Merge pull request #3215 from elizaOS/tcm-remove-duplicated-dependencies

fix: remove duplicated dependencies

shakkernerd committed:
Merge branch 'develop' into fix-quickintel-plugin

shakkernerd committed:
Merge pull request #3208 from azep-ninja/fix-quickintel-plugin

fix: quick-intel plugin optimizations & fixes.

tcm390 committed:
add ENABLE_TWITTER_POST_GENERATION config

tcm390 committed:
add ENABLE_TWITTER_POST_GENERATION config

tcm390 committed:
add ENABLE_TWITTER_POST_GENERATION

shakkernerd committed:
Merge pull request #3219 from elizaOS/tcm-twitter-disable-post

feat: Add configuration for enabling/disabling Twitter post generation

vince0656 committed:
refactor: update .env.example with discourse config vars and adjust tests

vince0656 committed:
Merge branch 'develop' into 2674-discourse-plugin

vince0656 committed:
refactor: spelling of environment file name

odilitime commented:
Hello,

We are changing our plugin development strategy to be more scalable. We have moved the plugins out into their own repos and we're looking for people to either maintain those or own them on their own Github.

If you'd like to be a maintainer,  file an issue in the plugin repo and join our Discord https://discord.gg/elizaos to coordinate.

If you'd like to control the plugin on your own Github, please add an issue to the plugin repo pointing to your repo, and add a modification to the registry. Submit a PR to edit the registry here: https://github.com/elizaos-plugins/registry

Closing this PR for now. Let us know if you have any questions.
